src/common/util.js

```diff
@@ -430,43 +431,57 @@ function getIndentSize(value, tabWidth) {
   );
 }
 
-function printString(raw, options, isDirectiveLiteral) {
+function getPreferredQuote(raw, preferredQuote) {
+  assert(
```

Comment:
Looks like we don’t use `assert` anywhere else. Maybe we can just remove this `assert` (no need to validate I think)

Reply:
removed